packages/prover/src/utils/rpc.ts

```diff
@@ -99,7 +101,6 @@ export class ELRpc {
   }
 
   getRequestId(): string {
-    // TODO: Find better way to generate random id
-    return (Math.random() * 100000000000000000).toFixed(0);
```

Comment:
In the current use case the `requestId` is not that important to be generated anyway. Even if we send same id every-time, it will still work fine as requests are processed sequentially. 



With incremental ids, the edge case will left is a long running proxy server. Though it will take really a long time to overflow the integer.

Reply:
> Even if we send same id every-time, it will still work fine as requests are processed sequentially.

According to https://github.com/ChainSafe/lodestar/pull/6115 sending same id twice will cause issues.


> With incremental ids, the edge case will left is a long running proxy server

Theoretically this is true but 2**53-1 is the maximum request id we can support which is enormous, we can never reach that number in reality.